pkg/render/monitor.go

```diff
@@ -142,6 +164,44 @@ func (mc *monitorComponent) alertmanager() *monitoringv1.Alertmanager {
 	}
 }
 
+func (mc *monitorComponent) alertmanagerSecret() *corev1.Secret {
+	// This secret will be mounted as the Alertmanager configuration file.
+	// TODO: We should use the native CR support for this instead of using a secret for it.
+	return &corev1.Secret{
+		TypeMeta: metav1.TypeMeta{Kind: "Secret", APIVersion: "v1"},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "alertmanager-calico-node-alertmanager",
+			Namespace: common.TigeraPrometheusNamespace,
+		},
+		Data: map[string][]byte{
+			"alertmanager.yaml": []byte(alertmanagerConfig),
```

Comment:
I haven't reviewed the whole PR but this alertmanager configuration is supposed to be user configurable. This is resource should be treated as something that the operator doesn't overwrite if one is already provided.

Reply:
OK. Probably I need to leave it unmanaged by the operator. Thanks for clarification this.

Reply:
PR updated to have the Alertmanager configuration secret in the Operator namespace and then copied to the Prometheus namespace. Any changes in the Operator namespace will be reflected in the Prometheus namespace.